### DISCUSSION TITLE
选择双面打印“openright,blankleft,twoside”后，部分段落之间间距过大，部分标题与段落之间间距过大，部分图片尺寸超过页脚

### DISCUSSION THREAD

**Discussion by ANDATD20** [nuaatug/nuaathesis#87] [Q&A] [answered]
![屏幕截图 2022-08-04 201758](https://user-images.githubusercontent.com/106942556/182850551-200fcef1-c192-49fc-a2e8-264fed77f0a3.png)
![屏幕截图 2022-08-04 201825](https://user-images.githubusercontent.com/106942556/182850561-abba7cf9-94b6-412e-bfb8-ce7ae33395a3.png)
![屏幕截图 2022-08-04 201855](https://user-images.githubusercontent.com/106942556/182850562-30a2c976-e0c0-4258-804a-4880074f6ada.png)

**Comment by yzwduck** [comment, marked as answer]
谢谢报告问题，确实是模板没有考虑到的地方。

我大概复现了同样的问题，验证了在 `\begin{document}` 前面加一句 `\raggedbottom` 能回复单面排版的间距。

我会将这句话加入到模板里的。
